src/test/java/io/jenkins/plugins/analysis/core/columns/IssueCounterITest.java

```diff
@@ -0,0 +1,94 @@
+package io.jenkins.plugins.analysis.core.columns;
+
+import static io.jenkins.plugins.analysis.core.testutil.Assertions.assertThat;
+
+import io.jenkins.plugins.analysis.warnings.Java;
+import java.io.IOException;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.DomNode;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import jenkins.model.Jenkins;
+
+import io.jenkins.plugins.analysis.core.testutil.IntegrationTestWithJenkinsPerTest;
+
+/**
+ * Integration Test for the Issue counter column.
+ */
+public class IssueCounterITest extends IntegrationTestWithJenkinsPerTest {
+
+    private static final String ISSUE_ELEMENT_XPATH = "/html/body/div[4]/div[2]/div[2]/div[2]/table/tbody/tr[2]/td[8]/a";
+    private static final String JAVAC_ONE_WARNING = "javac_1_warning.txt";
+    private static final String JAVAC_TEN_WARNINGS = "javac_10_warnings.txt";
+    private static final String RESULT_FILE_NAME = "javac.log";
+    private static final String JAVA_FILE_PATTERN = "**/*.log";
+
+    /**
+     * Tests that the issue counter is displayed correctly with 1 issue.
+     * @throws IOException Shouldn't throw.
```

Comment:
Remove `@throws` and extract the method that throws the exception. There you can catch it and rethrow a RTE.